src/pretix/base/views/metrics.py

```diff
@@ -0,0 +1,35 @@
+from django.conf import settings
+from http import HttpResponse
+from .. import metrics
```

Comment:
This import is neither correct (``http`` is inside django, not a top-level module) nor sorted the right way.